sfepy/discrete/fem/mappings.py

```diff
@@ -71,84 +192,85 @@ def get_physical_qps(self, qp_coors):
 
         return qps
 
-class VolumeMapping(FEMapping):
-    """
-    Mapping from reference domain to physical domain of the same space
-    dimension.
-    """
-
     def get_mapping(self, qp_coors, weights, poly_space=None, ori=None,
-                    transform=None):
+                    transform=None, is_face=False, extra=(None, None, None)):
         """
         Get the mapping for given quadrature points, weights, and
         polynomial space.
 
+        Parameters
+        ----------
+        qp_coors: numpy.ndarray
+            The coordinates of the integration points.
+        weights:
+            The integration weights.
+        poly_space: PolySpace instance
+            The PolySpace instance.
+        ori: numpy.ndarray
+            Element orientation, used by hierarchical basis.
+        transform: numpy.ndarray
+            The transformation matrix applied to the basis functions.
+        is_face: bool
+            Is it the boundary of a region?
+        extra: tuple
+            The extra data for surface derivatives:
+              - the derivatives of the field boundary base functions with
+                respect to the reference coordinates
+              - the boundary connectivity
+              - the derivatives of the domain boundary base functions with
+                respect to the reference coordinates
+
         Returns
         -------
-        cmap : CMapping instance
-            The volume mapping.
+        pycmap: PyCMapping instance
+            The domain mapping fields.
```

Comment:
'The domain mapping data.' to avoid mistaking it with fields?

Reply:
Yes, `mapping data` it is better.